rise-reveal/package.json

```diff
@@ -33,16 +33,17 @@
   ],
   "scripts": {
     "build": "for target in copy patch; do npm run $target; done",
-    "copy": "mkdir -p export; for dep in reveal.js reveal.js-chalkboard; do cp -r ./node_modules/$dep/ ./export/$dep/; done",
-    "patch": "for target in patch-reveal-css patch-notes patch-themes ; do npm run $target; done",
+    "copy": "mkdir -p export; for dep in reveal.js; do cp -r ./node_modules/$dep/ ./export/$dep/; done; cp -r ./node_modules/reveal.js-plugins/chalkboard ./export/reveal.js-chalkboard/",
+    "patch": "for target in patch-reveal-css patch-notes patch-themes patch-chalkboard ; do npm run $target; done",
     "patch-reveal-css": "sed -i.upstream '11 s_^_/*_' export/reveal.js/css/reveal.css",
     "patch-notes": "bash patch-notes-plugin.sh",
     "patch-themes": "bash patch-reveal-themes.sh",
+    "patch-chalkboard": "bash patch-chalkboard.sh",
     "clean": "rm -rf node_modules export"
   },
   "dependencies": {
-    "reveal.js": "~3.8.0",
-    "reveal.js-chalkboard": "~1.0.0"
+    "reveal.js": "~3.9.0",
+    "reveal.js-plugins": "rajgoel/reveal.js-plugins#3.9.0"
   },
   "version": "380.0.2"
```

Comment:
you'd need to change this as well; 380 was for 3.8.0

Reply:
going for 390.0.1 then